template.yaml

```diff
@@ -60,11 +74,26 @@ objects:
                 value: "7077"
               - name: SPARK_MASTER_WEBUI_PORT
                 value: "8080"
+              - name: NSS_WRAPPER_PASSWD
```

Comment:
I believe that the NSS_WRAPPER stuff is currently handled in our launch.sh script.